staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/controller.go

```diff
@@ -76,13 +77,28 @@ func NewAggregationController(downloader *aggregator.Downloader, openAPIAggregat
 }
 
 // Run starts OpenAPI AggregationController
-func (c *AggregationController) Run(stopCh <-chan struct{}) {
+func (c *AggregationController) Run(stopCh <-chan struct{}, initialAggrDoneCh chan<- struct{}) {
 	defer utilruntime.HandleCrash()
 	defer c.queue.ShutDown()
 
 	klog.Info("Starting OpenAPI AggregationController")
 	defer klog.Info("Shutting down OpenAPI AggregationController")
 
+	// Initially aggregate OpenAPI spec before running worker.
+	var wg sync.WaitGroup
+	for _, name := range c.openAPIAggregationManager.GetAPIServiceNames() {
```

Comment:
is this guaranteed to be after the informer has synchronized?

Reply:
same question as mine above